data/brands/amenity/cinema.json

```diff
@@ -546,6 +547,7 @@
         "brand:wikidata": "Q6484805",
         "brand:wikipedia": "en:Landmark Theatres",
         "name": "Landmark Theatres",
+        "operator: "Kinepolis", 
```

Comment:
I think you mixed [Landmark Theatres](https://en.wikipedia.org/wiki/Landmark_Theatres) with [Landmark Cinemas](https://en.wikipedia.org/wiki/Landmark_Cinemas). It looks like Landmark Cinemas isn't in NSI yet so feel free to add it
```suggestion
```